packages/web-components/src/components/va-link/va-link.tsx

```diff
@@ -141,19 +147,31 @@ export class VaLink {
       video,
       reverse,
       iconName,
-      iconSize
+      iconSize,
     } = this;
 
     const linkClass = classNames({
       'va-link--reverse': reverse,
-      'link--center': iconName
+      'link--center': iconName,
     });
 
+    const backArrow = (
+      <va-icon
+        icon="arrow_back"
+        class="link-icon--left link-icon--back"
+      ></va-icon>
```

Comment:
We have `part="icon"` on the other icon variations. Do we think teams might need it on this icon as well?

Reply:
I looked and we currently only have the `part="icon"` on the "custom icon" variant, so I was basing not including it here off of that.